README.md

```diff
@@ -206,6 +211,183 @@ try {
 }
 ```
 
+<a name="in-process-linking"/>
+
+## Trace in-process asynchronous execution
+
+You can use the SDK to trace asynchronous in-process code execution. This might be useful if OneAgent does not support the threading framework or specific asynchronous libraries. In-process-linking should be used to link other services (Database-, Webrequests, ...) between thread or queuing boundaries currently not supported out-of-the-box by OneAgent.
```

Comment:
why the "-" for "Database-"?
"queuing" -> "queueing"
also I'd personally write "the OneAgent" with an article